source/ocsp-support/ocsp-support.rst

```diff
@@ -134,6 +134,41 @@ Drivers SHOULD validate OCSP Responses in the manner specified in `RFC
 6960: 3.2 <https://tools.ietf.org/html/rfc6960#section-3.2>`__ to the
 extent that their TLS library allows.
 
+Suggested OCSP Caching Behavior
+-------------------------------
+Drivers with sufficient control over their TLS library's OCSP
+behavior SHOULD implement an OCSP cache. The key for this cache
+SHOULD be the certificate identifier (CertID) of the OCSP request
+as specified in `RFC 6960: 4.1.1
+<https://tools.ietf.org/html/rfc6960#section-4.1.1>`__.
+For convenience, the relevant section has been duplicated below\:
```

Comment:
Is is necessary to escape the colon? I thought RST only applies special formatting to the subsequent content if a line ends with `::`.

Reply:
Ah you're right: my mistake! Fixed.